### ISSUE TITLE
Could you give more information about your student model on pubfig65?

### ISSUE THREAD

**Issue by zoeleeee** [bolunwang/translearn#2]
I want to use your pre-trained strudent model in my experiments. I cannot find your training dataset and the test data file cannot be read by h5py normally. I have tried the dataset you metioned in readme, but  student model input need 224*224 images, while pubfig83 is 100*100, so I don't know how your model deal with transforming. So is it convenient for you to publish your pubfig65 dataset and give more details about your student model(e.g. label) or tell me how you transform images from 100*100 to 224*224?

**Comment by bolunwang** [comment]
What we did in our experiments was to upsample all images to 224*224 (cause we need to fit it into VGG-Face). This should be fairly simple to do when you load images. If you use Keras.preprocessing.image, here is the code snippet you can use to specify the resolution you want.

```
from keras.preprocessing import image
target_size = (224, 224)
img = image.load_img(img_path, target_size=target_size)
x = image.img_to_array(img)
```
